website/docs/expansion.md

```diff
@@ -1,97 +1,41 @@
 ---
 id: expansion
-title: Validation of Workload Resources
+title: Validating Workload Resources using `ExapansionTemplate`s
```

Comment:
```suggestion
title: Validating Workload Resources Using `ExapansionTemplate`s
```

Reply:
i think it's fine not to capitalize the `u` there